config.yml.example

```diff
@@ -0,0 +1,3 @@
+login: some_login
+password: some_password
+cluster_url: https://some_hostname:4242
```

Comment:
Wonder if we can avoid keeping the credentials in config.yml and try to leverage KUBECONFIG instead.

Reply:
I changed the playbook, now it's the same we have here https://github.com/fusor/ocp-velero-ansible/blob/master/roles/login_ocp/tasks/main.yml